libraries/SITL/SITL.cpp

```diff
@@ -403,6 +403,14 @@ Vector3f SITL::convert_earth_frame(const Matrix3f &dcm, const Vector3f &gyro)
     return Vector3f(phiDot, thetaDot, psiDot);
 }
 
+float SITL::get_rangefinder(uint8_t rot) {
+    if (rot < ROTATION_MAX) {
+        return state.rangefinder_m[rot];
+    } else {
+        return 0;
```

Comment:
This doesn't look great, 0 being in-band and all.

Could return NaN, could have a return-param, could abort(), ....

Reply:
Now returns -1 and new rangefinder backend sets it status to no data for negative numbers